### PR TITLE
created a Login Data in Backend

### PR DESCRIPTION
**Pull request by Starkiller1988**
created Spring Security 
Login with Token
shows in the MongoDB

### CODE REVIEW

File: backend/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,3 +1,8 @@
 server.port=${PORT:8080}
-spring.data.mongodb.uri=${MONGODB_URI:mongodb://localhost:27017/cards}
-spring.mvc.pathmatch.matching-strategy=ant_path_matcher
\ No newline at end of file
+spring.data.mongodb.uri=${MONGODB_URI:mongodb://localhost:27017}
```
Comment (gossie) [inline backend/src/main/resources/application.properties]: Why did you remove the database name for localhost?

File: backend/src/main/resources/application.properties [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+spring.mvc.pathmatch.matching-strategy=ant_path_matcher
+logging.file.path=/logs
+
+logging.level.de.neuefische.glossary=DEBUG
+
+app.jwt.secret=${APP_JWT_SECRET:myJWTToken}en}
```
Comment (gossie) [inline backend/src/main/resources/application.properties]: Where does "en}" come from?

File: backend/src/main/java/com/example/demo/LoginRegister/UserDTO.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package com.example.demo.LoginRegister;
```
Comment (gossie) [inline backend/src/main/java/com/example/demo/LoginRegister/UserDTO.java]: Why did you copy this class? It seems like it's never used.

File: backend/src/main/resources/application.properties [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
-spring.data.mongodb.uri=${MONGODB_URI:mongodb://localhost:27017}
+spring.data.mongodb.uri=${MONGODB_URI:mongodb://localhost:27017/users}
 spring.mvc.pathmatch.matching-strategy=ant_path_matcher
 logging.file.path=/logs
 
 logging.level.de.neuefische.glossary=DEBUG
 
-app.jwt.secret=${APP_JWT_SECRET:myJWTToken}en}
\ No newline at end of file
+app.jwt.secret=${APP_JWT_SECRET:myJWTToken}}
```
Comment (gossie) [inline backend/src/main/resources/application.properties]: There are still two `}`

**Review by gossie — APPROVED** [review]

File: backend/src/main/java/com/example/demo/LoginRegister/UserDTO.java (hunk 3 above)
Comment (Starkiller1988) [reply backend/src/main/java/com/example/demo/LoginRegister/UserDTO.java]: deleted it

File: backend/src/main/resources/application.properties (hunk 2 above)
Comment (Starkiller1988) [reply backend/src/main/resources/application.properties]: was a typo

File: backend/src/main/resources/application.properties (hunk 1 above)
Comment (Starkiller1988) [reply backend/src/main/resources/application.properties]: changed it
